changelogs/fragments/2246-terraform.yaml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - terraform - add a function which retrieves  the used terraform version (https://github.com/ansible-collections/community.general/pull/2246).
```

Comment:
```suggestion
```
I would not mention this, since it's not helpful to end-users and can be seen as an implementation detail of the next two entries.